tutorials/btp-app-hana-cloud-setup/btp-app-hana-cloud-setup.md

```diff
@@ -263,9 +262,9 @@ If you need to create an SAP HANA Cloud service instance first &rarr; continue w
     > ```
 
 
-[VALIDATE_1]
 [OPTION END]
 
+[VALIDATE_1]
```

Comment:
This code is obsolete. Please use v2 parser syntax.

Reply:
please see comment.